plugins/language_janet.lua

```diff
@@ -0,0 +1,685 @@
+-- Support for the Fennel programming language: https://fennel-lang.org
+-- Covers all the keywords up to Fennel version 0.10.0
+
+-- Currently only covers highlighting, not indentation, delimiter
+-- matching, or evaluation.
+
+local syntax = require "core.syntax"
+
+syntax.add {
+  files = "%.janet$",
+  comment = ";",
+  patterns = {
+    { pattern = "#.-\n",              type = "comment"  },
+    { pattern = { '"', '"', '\\' },   type = "string"   },
+    { pattern = "0x[%da-fA-F]+",      type = "number"   },
+    { pattern = "-?%d+[%d%.]*",       type = "number"   },
+    { pattern = "-?%.?%d+",           type = "number"   },
+    { pattern = "%f[^(][^()'%s\"]+",  type = "function" },
+    { pattern = ":[%w_-/]*",          type = "constant"   },
+    { pattern = "nil",                type = "literal"   },
```

Comment:
``nil``, ``true`` and ``false`` probably should be added to symbols field as they aren't patterns